setup.py

```diff
@@ -18,8 +18,8 @@
 name = 'mediatimestamp'
 description = 'A timestamp library for high precision nanosecond timestamps'
 url = 'https://github.com/bbc/rd-apmm-python-lib-mediatimestamp'
-author = 'James Sandford'
-author_email = 'james.sandford@bbc.co.uk'
+author = 'BBC R&D'
```

Comment:
Just a thought. Is `&` safe in this field?

Reply:
Good thought - https://www.python.org/dev/peps/pep-0345/ doesn't have anything to say on chars that are or aren't permitted, just that they have to be allowed by RFC822 fields, which seems to permit any ASCII char.
Let's see if PyPI gets on with it...